code/edgeconsent/src/androidTest/java/com/adobe/marketing/mobile/edge/consent/util/MonitorExtension.java

```diff
@@ -74,30 +65,11 @@ public void error(ExtensionError extensionError) {
 	public static void unregisterExtension() {
 		Event event = new Event.Builder(
 			"Unregister Monitor Extension Request",
-			TestConstants.EventType.MONITOR,
-			TestConstants.EventSource.UNREGISTER
+			ConsentTestConstants.EventType.MONITOR,
+			ConsentTestConstants.EventSource.UNREGISTER
 		)
 			.build();
-		MobileCore.dispatchEvent(
-			event,
-			new ExtensionErrorCallback<ExtensionError>() {
-				@Override
-				public void error(ExtensionError extensionError) {
-					MobileCore.log(LoggingMode.ERROR, LOG_TAG, "Failed to unregister Monitor extension.");
-				}
-			}
-		);
-	}
-
-	/**
-	 * Add an event to the list of expected events.
-	 * @param type the type of the event.
-	 * @param source the source of the event.
-	 * @param count the number of events expected to be received.
-	 */
-	public static void setExpectedEvent(final String type, final String source, final int count) {
```

Comment:
Looks like the ability to set an "expected" event was removed from the MonitorExtension. If this functionality isn't needed, then can you remove the `expectedEvents` variable and any handling of checking for or returning expected events?